config/default/jenkins.yaml

```
@@ -124,10 +206,7 @@ jenkins:
                     globalMatrix:
                       grantedPermissions:
                         - "Overall/Administer:release-core"
-                        - "Overall/Read:release-core-dev"
-                        - "Job/Build:release-core-dev"
-                        - "Job/Cancel:release-core-dev"
-                        - "Job/Read:release-core-dev"
+                        - "Overall/SystemRead:all"
```

Because of weirdness in permissions and how they applied you also need these:
```suggestion
                        - "Overall/SystemRead:all"
                        - "Overall/Read:all"
                        - "Job/Read:all"
```
Administer is the only god permission, everything else needs applying explicitly

I'm planning to try get a 'full read' role added in role-strategy-plugin which would make this simpler as you don't need to care about specific permissions it just grants all the read ones that make sense